symplyphysics/laws/electricity/circuits/law_of_three_second_for_vacuum_diode.py

```diff
@@ -0,0 +1,37 @@
+from sympy import Eq, solve
```

Comment:
Use descriptive names. Law of three second does not give any hint, what it is about.